src/Streamly/Internal/Data/Stream/IsStream/Common.hs

```diff
@@ -705,6 +729,8 @@ splitOnSeq
     => Array a -> Fold m a b -> t m a -> t m b
 splitOnSeq patt f m =
     IsStream.fromStreamD $ D.splitOnSeq patt f (IsStream.toStreamD m)
+-- splitOnSeq patt f =
+    -- foldManyPost (Fold.takeEndBySeq_ patt f)
```

Comment:
This will not work when the fold terminates right?

Reply:
Some input is lost?

Reply:
The problem will be with the last fold when the stream terminates before the fold.